content/docs/authorization/v2.0 Tech Preview/configuration/powerflex/_index.md

```diff
@@ -0,0 +1,117 @@
+---
+title: PowerFlex
+linktitle: PowerFlex
+description: >
+  Enabling CSM Authorization for PowerFlex CSI Driver
+---
+
+## Configuring PowerFlex CSI Driver with CSM for Authorization
+
+After CSM Authorization is installed and a tenant has been created, the [dellctl CLI](../../../../references/cli/) can be used to generate a token for that tenant.
+
+1. Generate the token for the desired tenant that will be used for Authorization.
+   
+   ```console
+    dellctl generate token --addr csm-authorization.com:<port> --insecure true --tenant <tenant> --access-token-expiration 30m0s --refresh-token-expiration 1480h0m0s | k -n vxflexos apply -f -
```

Comment:
Relating to https://github.com/dell/csm-docs/pull/1101/files#r1610186172, the storage administrator generates the token and is delivered to the k8s administator.